i18n/de/strings.xml

```diff
@@ -13,4 +13,7 @@
   <!-- XTXT: Booster Notification Rule: BNR-DE-0210, STIKO-Empfehlung Optimierung Grundimmunisierung Janssen; Eintritt  -->
   <string name="BNR-DE-0210">"Die Ständige Impfkommission (STIKO) empfiehlt allen Personen eine weitere Impfstoffdosis zur Optimierung der Grundimmunisierung, die mit einer Dosis des Janssen-Impfstoffs (Johnson &amp; Johnson) grundimmunisiert wurden, bei denen keine Infektion mit dem Coronavirus SARS-CoV-2 nachgewiesen wurde und wenn ihre Janssen-Impfung über 4 Wochen her ist.\n\nDa Sie laut Ihrer gespeicherten Zertifikate dieser Personengruppe angehören und noch keine weitere Impfung erhalten haben, möchten wir Sie auf diese Empfehlung hinweisen. (Regel DE-BNR-0210)\n\nDieser Hinweis basiert ausschließlich auf den auf Ihrem Smartphone gespeicherten Zertifikaten. Die Verarbeitung der Daten erfolgte auf Ihrem Smartphone. Es wurden hierbei keine Daten an das RKI oder Dritte übermittelt."</string>
 
+  <!-- XTXT: Booster Notification Rule: BNR-DE-0400, STIKO-Empfehlung 2. Booster 60 Jahre; Eintritt  -->
+  <string name="BNR-DE-0400">"Die Ständige Impfkommission (STIKO) empfiehlt allen Personen eine zweite Auffrischimpfung, die mindestens 60 Jahre alt sind und deren erste Auffrischimpfung mehr als 6 Monate her ist. Diese Empfehlung gilt auch für Personen, die nach ihrer Grundimmunisierung eine Corona-Infektion durchgemacht haben, wenn diese Infektion mehr als 6 Monate her ist.\n\nDa Sie laut Ihrer gespeicherten Zertifikate einer dieser Personengruppen angehören und noch keine zweite Auffrischimpfung erhalten haben, möchten wir Sie auf diese Empfehlung hinweisen. (Regel DE-BNR-0400)\n\nDieser Hinweis basiert ausschließlich auf den auf Ihrem Smartphone gespeicherten Zertifikaten. Die Verarbeitung der Daten erfolgte auf Ihrem Smartphone. Es wurden hierbei keine Daten an das RKI oder Dritte übermittelt.\n\nBitte beachten Sie, dass Sie diesen Hinweis aus technischen Gründen auch dann erhalten, wenn Sie eine dritte Impfdosis erhalten haben, die wegen einer vorausgegangenen Infektion, die aber nicht mittels eines Genesenenzertifikats auf Ihrem Smartphone dokumentiert ist, bereits als zweite Auffrischimpfung zählt. Sie können diesen Hinweis dann ignorieren. Falls Sie unsicher sind, lassen Sie sich zum weiteren Vorgehen bitte ärztlich beraten."</string>
```

Comment:
```suggestion
  <string name="BNR-DE-0400">"Die Ständige Impfkommission (STIKO) empfiehlt allen Personen eine zweite Auffrischimpfung, die mindestens 60 Jahre alt sind und deren erste Auffrischimpfung mehr als 6 Monate her ist. Diese Empfehlung gilt auch für Personen, die nach ihrer Grundimmunisierung eine Corona-Infektion durchgemacht haben, wenn diese Infektion mehr als 6 Monate her ist.\n\nDa Sie laut Ihrer gespeicherten Zertifikate einer dieser Personengruppen angehören und noch keine zweite Auffrischimpfung erhalten haben, möchten wir Sie auf diese Empfehlung hinweisen. (Regel DE-BNR-0400)\n\nDieser Hinweis basiert ausschließlich auf den auf Ihrem Smartphone gespeicherten Zertifikaten. Die Verarbeitung der Daten erfolgte auf Ihrem Smartphone. Es wurden hierbei keine Daten an das RKI oder Dritte übermittelt.\n\nBitte beachten Sie, dass Sie diesen Hinweis aus technischen Gründen auch dann erhalten, wenn Sie eine dritte Impfdosis erhalten haben, die wegen einer vorausgegangenen Infektion, welche allerdings nicht mittels eines Genesenenzertifikats in der Corona-Warn-App dokumentiert ist, bereits als zweite Auffrischimpfung zählt. Sie können diesen Hinweis dann ignorieren. Falls Sie unsicher sind, lassen Sie sich zum weiteren Vorgehen bitte ärztlich beraten."</string>
```

Recommendation to make the text clearer. Also don't write that something isn't on the smartphone but rather not in the CWA, as you have no idea and can't tell whether the user has the certificate e.g. stored in the CovPass app on their smartphone or as a PDF somewhere.

Reply:
Hi @Ein-Tim - these are valid points.

The text has been agreed upon by BMG/RKI (in a quite lenghty process), however, and is aligned with the wording that has been used in the older BNRs. 

We will keep your suggestions in mind for future changes to the BNRs.

Reply:
Thank you Britta!

Reply:
The suggested text change referring explicitly to "Corona-Warn-App" would be problematic since the texts are used generically in both CWA and CovPass.

Reply:
@MikeMcC399 In this repos README it says:
> This repository contains the description texts for the booster notification rules to be displayed in the Corona-Warn-App.

So this should not be a problem as this is only the translation & repo for the CWA.

Reply:
Though this repo belongs to the CWA project, we only implement one set of rules that is used by CovPass as well.

Reply:
@BrittaTSI 

Maybe it would make sense to add this to the README?